package.json

```diff
@@ -44,7 +44,7 @@
         "@kobalte/tailwindcss": "^0.5.0",
         "@modular-forms/solid": "^0.18.0",
         "@mutinywallet/barcode-scanner": "5.0.0-beta.3",
-        "@mutinywallet/mutiny-wasm": "0.4.4",
+        "@mutinywallet/mutiny-wasm": "0.4.5",
```

Comment:
Needs updating for the package versioning too

Reply:
nice catch